packages/web-components/src/menu/menu.stories.ts

```diff
@@ -0,0 +1,230 @@
+        </fluent-menu>
+      </fluent-menu-item>
+      <fluent-menu-item class="header">Checkbox Group Header</fluent-menu-item>
+      <fluent-menu-item role="menuitemcheckbox" ?disabled=${x => x.disabled}>
+        Option 1
+        <span slot="start">${Cut20Filled}</span>
+      </fluent-menu-item>
+      <fluent-menu-item role="menuitemcheckbox" ?disabled=${x => x.disabled}>
+        Option 2
+        <span slot="start">${Edit20Filled}</span>
+      </fluent-menu-item>
+      <fluent-menu-item role="menuitemcheckbox" ?disabled=${x => x.disabled}>
+        Option 3
+        <span slot="start">${ClipboardPaste20Filled}</span>
+      </fluent-menu-item>
+      <fluent-menu-item class="header">Radio Group Header</fluent-menu-item>
+      <fluent-menu-item role="menuitemradio" ?disabled=${x => x.disabled}>
+        Option 1
+        <span slot="start">${Cut20Filled}</span>
+      </fluent-menu-item>
+      <fluent-menu-item role="menuitemradio" ?disabled=${x => x.disabled}>
+        Option 2
+        <span slot="start">${Edit20Filled}</span>
+      </fluent-menu-item>
+      <fluent-menu-item role="menuitemradio" ?disabled=${x => x.disabled}>
+        Option 3
+        <span slot="start">${ClipboardPaste20Filled}</span>
+      </fluent-menu-item>
+    </fluent-menu>
+  </div>
+`;
+
+export default {
+  title: 'Components/Menu',
+  args: {
+    disabled: false,
+  },
+  argTypes: {
+    disabled: {
+      description: 'Disables Menu item',
+      table: {
+        defaultValue: { summary: false },
+      },
+      control: 'boolean',
+      defaultValue: false,
+    },
+  },
+} as MenuStoryMeta;
+
+export const Menu = renderComponent(storyTemplate).bind({});
+
+export const MenuWithCheckboxes = renderComponent(html<MenuStoryArgs>`
+  <div style="width: 128px; position: relative;">
+    <fluent-menu>
+      <fluent-menu-item role="menuitemcheckbox">
+        Option 1
+        <span slot="start">${Cut20Filled}</span>
+      </fluent-menu-item>
+      <fluent-menu-item role="menuitemcheckbox">
+        Option 2
+        <span slot="start">${Edit20Filled}</span>
+      </fluent-menu-item>
+      <fluent-menu-item role="menuitemcheckbox">
+        Option 3
+        <span slot="start">${ClipboardPaste20Filled}</span>
+      </fluent-menu-item>
+    </fluent-menu>
+  </div>
+`);
+
+export const MenuWithRadios = renderComponent(html<MenuStoryArgs>`
+  <div style="width: 128px; position: relative">
+    <fluent-menu>
+      <fluent-menu-item role="menuitemradio">
+        Option 1
+        <span slot="start">${Cut20Filled}</span>
+      </fluent-menu-item>
+      <fluent-menu-item role="menuitemradio">
+        Option 2
+        <span slot="start">${Edit20Filled}</span>
+      </fluent-menu-item>
+      <fluent-menu-item role="menuitemradio">
+        Option 3
+        <span slot="start">${ClipboardPaste20Filled}</span>
+      </fluent-menu-item>
+    </fluent-menu>
+  </div>
+`);
+
+export const MenuWithSubmenu = renderComponent(html<MenuStoryArgs>`
```

Comment:
On the first render, the submenu is positioned incorrectly:
![image](https://user-images.githubusercontent.com/9615899/220399051-0bea8311-061c-433f-b860-3bcd0b326903.png)

Reply:
@chrisdholt

Reply:
@brianchristopherbrady I don't repro this problem in our Foundation testing storybook: 
<img width="744" alt="image" src="https://user-images.githubusercontent.com/13071055/220775490-4b2b116e-1159-4ddf-80e7-6c71ccdb8939.png">

It's likely a nuanced delta in how you are styling it. If that's an issue, we can look at alternatives to resolve but let's see if we need to.

Reply:
@chrisdholt resolved by adding..

_menu-item.styles.ts:_

`:host { contain: layout; }`